ocrd_eval.md

```diff
@@ -7,36 +7,28 @@ evaluation metrics and evaluation tools that need to work together in a
 well-defined manner to allow users to make informed decisions about which OCR
 solution works best for their use case.
 
-## Evaluation metrics
+## Evaluation Metrics
 
 The evaluation of the success (accuracy) of OCR is a complex task for which multiple methods and metrics are available. It aims to capture quality in different aspects, such as the recognition of text, but also the detection of layout, for which different methods and metrics are needed.
 
 Furthermore, the time and resources required for OCR processing also have to be captured. Here we describe the metrics that were selected for use in OCR-D, how exactly they are applied, and what was the motivation.
 
-### Scope of these Definitions
+### Scope of These Definitions
```

Comment:
Why is `These` capitalizaed but `of` lower-case?

Reply:
I used https://titlecaseconverter.com/ with Chicago style. If you capitalize `these` or not depends on the style you choose. I just wanted to make the doc a bit more consistent since I used this style myself but didn't pay attention to what other added before this commit.

Feel free to object :)

Reply:
Because of [English capitalization rules](https://www.chicagomanualofstyle.org/16/ch08/ch08_sec157.html).

Reply:
I see, you learn something new everyday :) Thanks.